src/main/java/com/atlassian/bitbucket/jenkins/internal/client/WebhookRegisterRequest.java

```diff
@@ -0,0 +1,68 @@
+package com.atlassian.bitbucket.jenkins.internal.client;
+
+import com.atlassian.bitbucket.jenkins.internal.model.BitbucketWebhookRequest;
+
+import java.util.LinkedHashSet;
+import java.util.Set;
+
+import static java.util.Collections.unmodifiableSet;
+import static java.util.Objects.requireNonNull;
+import static org.codehaus.groovy.runtime.InvokerHelper.asList;
+
+/**
+ * Represents a webhook register request.
+ */
+public class WebhookRegisterRequest {
+
+    private final Set<String> events;
+    private final String url;
+    private final String name;
+
+    private WebhookRegisterRequest(WebhookRegisterRequestBuilder builder) {
+        events = unmodifiableSet(builder.events);
+        url = builder.url;
+        name = builder.name;
+    }
+
+    public BitbucketWebhookRequest getRequestPayload() {
+        return new BitbucketWebhookRequest(name, events, url, true);
+    }
+
+    /**
+     * A builder to provide fluent way of building webhook register request. 
+     */
+    public static class WebhookRegisterRequestBuilder {
```

Comment:
`WebhookRegisterRequestBuilder` -> `Builder`?

Reply:
I have deleted this class and used the model class for request